homeassistant/components/elgato/config_flow.py

```diff
@@ -55,8 +56,12 @@ async def async_step_zeroconf(
         if user_input is None:
             return self.async_abort(reason="connection_error")
 
-        # Hostname is format: my-ke.local.
-        host = user_input["hostname"].rstrip(".")
+        ip_address = user_input["host"]
+        try:
+            host = socket.gethostbyaddr(ip_address)
+        except:  # noqa: E722 pylint: disable=bare-except
```

Comment:
There is no reason to handle the exception this widely.

Reply:
Sorry, I was lazy...